bottlecap/src/proc/mod.rs

```diff
@@ -192,6 +352,21 @@ mod tests {
         safe_path.to_str().unwrap().to_string()
     }
 
+    #[test]
+    #[allow(clippy::float_cmp)]
```

Comment:
instead of float_cmp, you can use a comparison with < than delta. It should be safer and avoid this exception